WordPress/Classes/Services/MediaService.m

```diff
@@ -68,10 +68,12 @@ - (Media *)createMediaWith:(id<ExportableAsset>)exportable
         [self.managedObjectContext obtainPermanentIDsForObjects:@[media] error:nil];
         [self.managedObjectContext save: nil];
     }];
-
+    NSManagedObjectID *mediaObjectID = media.objectID;
     [self.managedObjectContext performBlock:^{
         // Setup completion handlers
         void(^completionWithMedia)(Media *) = ^(Media *media) {
+            media.remoteStatus = MediaRemoteStatusLocal;
```

Comment:
Does this mean we'll go created -> processing -> local -> pushing -> sync?

Reply:
Yep, this could be useful for offline mode or in moment where we don't have connectivity.